ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -189,58 +189,55 @@ StatusSectionLayout {
         return root.chatContentModule.chatDetails.isUsersListAvailable
     }
 
-    rightPanel: Component {
```

Comment:
Any prob with this not being a Component? :thinking:

Reply:
I don't see any. What we have in the qt5 layout is a Control with a Loader for this component that's always active. The `showRightPanel` flag controls the Control opacity and visibility.

With qt6 we'll have an Item as input for the rightPanel. If the `showRightPanel` flag is false it will behave in a similar way, hiding it. The code is almost identical, just replacing an always active `Loader` with `LayoutItemProxy`